recipes/cruft/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cruft" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 300a2f19dfd94e44a66f5b68d179a2a57b03c2fc7bbc25367d5b5f5c26681e0f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry >=0.12
+  run:
+    - python >=3.6
+    - cookiecutter >=1.6
+    - gitpython >=3.0
+    - toml >=0.10
+    - typer >=0.3.1
+    - click >=7.1.2
+
+test:
+  requires:
+    - poetry
+    - pytest >=5.0
+    - pytest-xdist >=1.34.0
+  commands:
+    - poetry run pytest -s -n auto
```

Comment:
We don't bundle the test dir with the package, so not sure how this is going to work. The best way to verify if everything is correct is to possibly run cruft and each of its sub commands with a help flag.

Reply:
typically in conda-forge we run `import` tests to check for existence and then test clis using the `-h` or `--help` flag:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#test

As @samj1912 points out, the tests are missing from the source distribution, so unless we switch the source to the github tarbal, you can't run them. I'm not necessarily recommending that. 

Instead, I think the section should look something like:

```
test:
   imports:
      - cruft
      - cruft.commands
   commands:
      - cruft -h
```

I know there are a bunch of cli subcommands and those should be checked if they support either the `-h` or `--help` flag.